File: NARESH_it/1_C_class/1_C_ENcryption_lab/27-AUG-24/2.c
/**
 * @Author: Dastan Alam
 * @Date:   2024-09-07 01:38:33 PM   13:09
 * @Last Modified by:   Dastan Alam
 * @Last Modified time: 2024-09-07 02:05:54 PM   14:09
 */
#include <stdio.h>
#include <stdlib.h>
#include <stdint.h>
#include <string.h>

int main() 
{ 
    // int a = 1, b = 4, c = 10; 
    // c = a = 10 == b + 6;
    // printf("%d %d %d", a, b, c);
    // return 0;
    { 
        int a = 1, b;
        // b = ++a + a++ + a++; 
        // printf("a=%d, b=%d\n", a, b); 
        a = 1;
        b = a++ + ++a ;
        //   1  + 3 
        printf("a=%d, b=%d", a, b); 
        return 0; 
    }
}

File: VS Encrytions cloud/2022/GO Encrypt/2.go
/**
 * @Author: Dastan Alam
 * @Date:   13-03-2022  01:49:48 PM  13:03:00
 * @Last Modified by:   Dastan Alam
 * @Last Modified time: 13-03-2022  01:52:51 PM  13:03:40
 */
package main

import "fmt"

func main() {
	var b int = 15
	var a int
	numbers := [6]int{1, 2, 3, 5}

	/* for loop execution */
	for a := 0; a < 10; a++ {
		fmt.Printf("value of a: %d\n", a)
	}
	for a < b {
		a++
		fmt.Printf("value of a: %d\n", a)
	}
	for i, x := range numbers {
		fmt.Printf("value of x = %d at %d\n", x, i)
	}
}
